auto_trainer/function.yaml

```diff
@@ -138,14 +138,38 @@ spec:
         default: null
       outputs:
       - default: ''
-      lineno: 267
+      lineno: 283
   description: Automatic train, evaluate and predict functions for the ML frameworks
     - Scikit-Learn, XGBoost and LightGBM.
   default_handler: train
   disable_auto_mount: false
   env: []
-  priority_class_name: ''
+  resources:
+    requests:
+      memory: 1Mi
+      cpu: 25m
+    limits:
+      memory: 20Gi
+      cpu: '2'
+  priority_class_name: igz-workload-medium
   preemption_mode: prevent
-  affinity: null
+  affinity:
```

Comment:
I don't think it should have an affinity. It would not always run on eks

Reply:
also remove resources and priority class names

Reply:
fixed